framework/src/modules/http_api/defaults/config.js

```diff
@@ -201,6 +222,13 @@ const defaultConfig = {
 				whiteList: ['127.0.0.1'],
 			},
 		},
+		apm: {
+			enabled: true,
+			options: {
+				name: 'Lisk-APM',
+				uriPath: '/lisk-stats',
```

Comment:
`lisk-stats` is generic and could be used in some other purpose in the future.
Maybe `http-stats`?